R/fda-table_32.R

```diff
@@ -0,0 +1,69 @@
+# Pre-processing
+
+library(tidyverse)
+library(scda)
+library(tern)
+
+# Safety population
+adsl <-
+  scda::synthetic_cdisc_dataset("rcd_2022_06_27", "adsl") %>%
+  tern::df_explicit_na() %>%
+  dplyr::filter(SAFFL == "Y") %>%
+  dplyr::select(STUDYID, USUBJID, SAFFL, ARM, ACTARM)
+
+# Vital signs - DIABP
+advs <-
+  scda::synthetic_cdisc_dataset("rcd_2022_06_27", "advs") %>%
+  tern::df_explicit_na() %>%
+  dplyr::filter(PARAMCD == "DIABP" & SAFFL == "Y" & AVISITN >= 1) %>%
+  dplyr::select(STUDYID, USUBJID, ARM, ACTARM, PARAMCD, PARAM, AVAL, AVALU, AVISITN, AVISIT) %>%
+  dplyr::group_by(USUBJID, PARAMCD) %>%
+  dplyr::mutate(maxDIABP = max(AVAL)) %>%
+  ungroup() %>%
+  dplyr::mutate(
+    L60 = (maxDIABP < 60),
+    U60 = (maxDIABP > 60),
+    U90 = (maxDIABP > 90),
+    U110 = (maxDIABP > 110),
+    UE120 = (maxDIABP >= 120)
+  ) %>%
+  formatters::var_relabel(
+    L60 = "<60",
+    U60 = ">60",
+    U90 = ">90",
+    U110 = ">110",
+    UE120 = ">=120"
+  )
+
+# Build layout
+lyt <-
+  rtables::basic_table(show_colcounts = TRUE) %>%
+  rtables::split_cols_by("ARM") %>%
+  tern::count_patients_with_flags(
+    var = "USUBJID",
+    flag_variables = var_labels(advs[, c("L60", "U60", "U90", "U110", "UE120")]),
+    table_names = "diabp"
```

Comment:
Adding a table name is usually not necessary unless you are using the same function multiple times in one table (to avoid duplicated `table_names`).
```suggestion
    flag_variables = var_labels(advs[, c("L60", "U60", "U90", "U110", "UE120")])
```